### PR TITLE
VXFM-1691 get additional properties for Brownfield.

### PR DESCRIPTION
**Pull request by aroberson**

### CODE REVIEW

**Comment by aroberson** [comment]
Do not resolve Jira

File: bin/discovery.rb [hunk 1]
```diff
@@ -267,24 +279,34 @@ def collect_distributed_switch_attributes(obj, parent)
...
+    detail = v.spec.ip.ipAddress if v && v.spec && v.spec.ip && v.spec.ip.ipAddress
+    detail
+  end
+
   default_response = {
-    :active_uplinks => active_uplinks,
-    :standby_uplinks => standby_uplinks
+      :active_uplinks => active_uplinks,
+      :standby_uplinks => standby_uplinks,
+      :host_ip_addresses => hostIps.compact
```
Comment (gavin-scott) [inline bin/discovery.rb]: Should have two-space indent here. IntelliJ screws it up.

File: bin/discovery.rb [hunk 2]
```diff
@@ -180,6 +186,12 @@ def collect_host_vmk_ips(host)
   virtual_nic_ip_array = host_virtual_nic_array.map { |hv_nic| hv_nic[:spec][:ip][:ipAddress] }
 end
 
+def collect_host_pnic_mac(host)
+  physical_nic_mac_array = host.config.network.pnic.map { |pnic| pnic[:mac] }
```
Comment (gavin-scott) [inline bin/discovery.rb]: Can you check what happens if the host is offline? Sometimes we have issues where some of the host-related calls hang if the host is in a bad state. You can check by powering off an ESXi server (that nobody is using!) in your vcenter and running this script against it.

Comment (aroberson) [reply bin/discovery.rb]: @gavin-scott There has been no issue running inventory against disconnected hosts.  Ran it against two different vcenters.

**Pull request merged**